src/lightning/pytorch/core/hooks.py

```diff
@@ -528,44 +461,19 @@ def val_dataloader(self) -> EVAL_DATALOADERS:
         - :meth:`setup`
 
         Note:
-            Lightning adds the correct sampler for distributed and arbitrary hardware
+            Lightning tries to add the correct sampler for distributed and arbitrary hardware
             There is no need to set it yourself.
 
-        Return:
-            A :class:`torch.utils.data.DataLoader` or a sequence of them specifying validation samples.
-
-        Examples::
-
-            def val_dataloader(self):
-                transform = transforms.Compose([transforms.ToTensor(),
-                                                transforms.Normalize((0.5,), (1.0,))])
-                dataset = MNIST(root='/path/to/mnist/', train=False,
-                                transform=transform, download=True)
-                loader = torch.utils.data.DataLoader(
-                    dataset=dataset,
-                    batch_size=self.batch_size,
-                    shuffle=False
-                )
-
-                return loader
-
-            # can also return multiple dataloaders
-            def val_dataloader(self):
-                return [loader_a, loader_b, ..., loader_n]
-
         Note:
             If you don't need a validation dataset and a :meth:`validation_step`, you don't need to
             implement this method.
-
-        Note:
```

Comment:
Removed these notes because this is now an error: https://github.com/Lightning-AI/lightning/pull/16837